.changeset/lucky-humans-talk.md

```diff
@@ -0,0 +1,19 @@
+---
+"@aws-amplify/ui-angular-mono": patch
```

Comment:
Should not be included in changeset, angualr is covered by line 16

Reply:
oh interesting, this file was autogenerated from changeset-bot.
removed `ui-angular-mono`